editor/scene_tree_dock.cpp

```diff
@@ -3104,7 +3104,23 @@ void SceneTreeDock::set_edited_scene(Node *p_scene) {
 	edited_scene = p_scene;
 }
 
+bool is_same_selection(const Vector<Node *> &p_first, const List<Node *> &p_second) {
+	if (p_first.size() != p_second.size()) {
+		return false;
+	}
+	for (Node *node : p_second) {
+		if (!p_first.has(node)) {
+			return false;
+		}
+	}
+	return true;
+}
+
 void SceneTreeDock::set_selection(const Vector<Node *> &p_nodes) {
+	// If the nodes selected are the same independently of order then return true.
```

Comment:
```suggestion
	// If the nodes selected are the same independently of order then return early.
```
No value is returned, or was this meant for the method above?

Reply:
yep this was meant for the above but maybe this is more usefull here with your modification